pom.xml

```diff
@@ -103,6 +103,10 @@
                 <scope>import</scope>
             </dependency>
             <dependency>
```

Comment:
@muuki88 you have an error in this `<dependency>` section which breaks build entirely.